### ISSUE TITLE
Getting not eligible for 30 days message with all results green

### ISSUE THREAD

**Issue by cgoulart35** [rcmaehl/WhyNotWin11#734]
Hello,

My machine is Windows 11 compatible it seems as all results are green below, but I receive the message; "Your Computer is ready for Windows 11 and its updates, but Windows Update may think you are not for 30 days. You can fix this by using the Windows Installation Assistant".

<img width="605" alt="image" src="https://github.com/rcmaehl/WhyNotWin11/assets/36514584/e12a9609-b20c-4d4d-86ca-7c844ccdb16b">

Upon seeing the message I checked PC Health Check and Windows Update, and also tried the Windows Installation Assistant. PC Health Check shows the generic message screenshotted below, and Windows Update says I am not eligible as expected. However, in my case, the Windows Installation Assistant is also saying my machine is not eligible. Am I missing something?

![image](https://github.com/rcmaehl/WhyNotWin11/assets/36514584/2ae696a1-f85d-4c66-b62d-56aba0833da4)

<img width="724" alt="image" src="https://github.com/rcmaehl/WhyNotWin11/assets/36514584/dc629fa6-7cec-42ac-a4d2-f026701f6dab">

**Comment by rcmaehl** [comment]
Your bootloader may be on another drive or another issue preventing a direct upgrade to Windows 11. I'd create an install USB and see if it'll let you upgrade that way

**Comment by cgoulart35** [comment]
I have the feeling something is wrong - I have installed Windows 11 installer to a USB and had to run the setup.exe file to upgrade an existing installation. I have tried to run it several times and every time it gets stuck on "Making sure you're ready to install" with 0% CPU usage. It has also closed by itself sometime leading me to believe it crashed.

I am not really sure where to go from here - I can try the .iso installation instead.
